samples/react/telehealth/README.md

```diff
@@ -41,8 +31,7 @@ Follow the steps to run the app locally.
    yarn
    ```
 
-1. Copy the `samples/.env.example` file as `samples/.env` and paste your Publish and Subscribe Keys
-   there.
+1. Copy the `samples/.env.example` file as `samples/.env` and paste your Publish and Subscribe Keys there.
 
 1. Go to the `samples/react/telehealth` folder.
 
```

Comment:
Could we also add similar info here? https://github.com/pubnub/chat-components-ios-examples/tree/master/telehealth-example#usernames

Reply:
Let's also link to https://www.pubnub.com/tutorials/cross-platform-chat-application-telehealth-react/

Reply:
I added links to tutorials in a README for each of the sample apps having a tutorial, as well as to the main repo README that has the table with links to all apps

Reply:
And I don't think the note about the `.env` file is applicable to the iOS repository, they probably use some other method

Reply:
@Salet I meant this part: <img width="1126" alt="Screenshot 2022-10-27 at 15 27 49" src="https://user-images.githubusercontent.com/40655785/198297397-0176afe2-e9a6-40c5-9947-dd0e6ef67544.png">

Reply:
This will be added to the branch containing changes to the telehealth app